File: examples/springapp/java-webflux/src/main/java/com/example/demo/MyApplication.java
package com.example.demo;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@SpringBootApplication
@RestController
public class MyApplication {

    @GetMapping
    String index() {
        return "hello, world";
    }

    public static void main(String[] args) {
        SpringApplication.run(MyApplication.class, args);
    }
}

File: examples/springapp/kotlin-webflux/src/main/kotlin/com/example/demo/MyApplication.kt
package com.example.demo

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Bean
import org.springframework.web.reactive.function.server.*

@SpringBootApplication
class MyApplication {
    @Bean
	fun http() = coRouter {
		GET("/") {
			ServerResponse.ok().bodyValueAndAwait("hello, world")
		}
	}
}

fun main(args: Array<String>) {
	runApplication<MyApplication>(*args)
}